staging/src/k8s.io/kubectl/docs/book/pages/container_debugging/copying_container_files.md

```diff
@@ -44,7 +44,7 @@ Copy a remote file from a Pod to a local file.
 {% sample lang="yaml" %}
 
 ```bash
-kubectl cp /tmp/foo <some-pod>:/tmp/bar
+kubectl cp <some-pod>:/tmp/foo /tmp/bar
```

Comment:
Ref. https://github.com/kubernetes/kubernetes/issues/79764 this will not work (and I can confirm), since the path part of the pod-spec starts with a leading `/`. What about just using `<some-pod>:foo` instead?

cc @apelisse

Reply:
But still a good catch @oke-py ! 😄

Reply:
It's just a warning of tar comand. Successful replication regardless of whether absolute paths are used or not.But I just suggest to remove the wrong message,which can be misleading.Modifying documents is another solution.